deploy/crs/kieapp_rhdm_authoring.yaml

```diff
@@ -2,5 +2,6 @@ apiVersion: app.kiegroup.org/v2
 kind: KieApp
 metadata:
   name: rhdm-authoring
+  annotations:
```

Comment:
Why this? Did you find annotations needed to be there or something?

Reply:
cleaned, I need more vigilance.